pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ejs

```diff
@@ -26,7 +26,7 @@
                         <div class="col-auto">
                             <span class="text-white">Course instances</span>
                         </div>
-                        <% if ((authz_data.has_course_permission_edit) && (! course.options.isExampleCourse)) { %>
+                        <% if ((authz_data.has_course_permission_edit) && (! course.options.isExampleCourse) && (! locals.needToSync)) { %>
```

Comment:
Is `locals` a variable in the current scope? It seems to me that this should die?

Reply:
Umm... yes? See lines 9, 28, 83 of `pages/partials/navbarInstructor.ejs`, for example.